objects.js

```diff
@@ -8496,17 +8497,19 @@ WatcherMorph.prototype.userMenu = function () {
         );
         if (this.currentValue &&
                 (isString(this.currentValue) || !isNaN(+this.currentValue))) {
-            menu.addItem(
-                'export...',
-                function () {
-                    var ide = myself.parentThatIsA(IDE_Morph);
-                    ide.saveFileAs(
-                        myself.currentValue.toString(),
-                        'text/plain;charset=utf-8',
-                        myself.getter // variable name
-                    );
-                }
-            );
+            menu.addItem('export...', this.valueExporter('plain'));
+        } else if (this.currentValue instanceof List) {
+            subMenu = new MenuMorph(this.currentValue);
+            if (!this.currentValue.contents.some(
```

Comment:
We can do that, but this is the only place where we're checking if a list is one-dimensional. Smells like over-engineering to me.

As for CSV and plain text being the same, see the next comment.